src/imdb_detector.py

```diff
@@ -86,6 +87,16 @@ def get_column_specific_label_mapping(self) -> dict:
             "md5sum": set_all_labels_to_ocr,
             "name": set_all_labels_to_ocr, # we checked manually, all unique values in this column are due to OCR errors
         }
+    
+    def _label_identical_columns(self):
```

Comment:
Here you are not labeling columns that are identical but that should be identical, right? So how about renaming this to `_label_columns_that_should_be_identical(self)`?